ui/components/multichain/notifications-settings-box/index.scss

```diff
@@ -0,0 +1,4 @@
+.notifications-settings-box__toggle {
+  width: 40px;
```

Comment:
Just want to confirm, do other places use fixed widths on extension? Or should we make areas got/shrink.

Rather than a fixed width, should this instead be a min-width?